.github/workflows/integration-test-object-suite.yaml

```
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions: ["v1.24.17", "v1.29.1"]
+        kubernetes-versions: ["v1.25.16", "v1.29.1"]
```

```suggestion
        kubernetes-versions: ["v1.25.16", "v1.29.x"]
```

upgrade 1.29 version if available